src/docker-utils.coffee

```diff
@@ -90,6 +91,7 @@ do ->
 							.on('id', resolve)
 							.on('error', reject)
 					.on 'error', reject
+				.timeout(DELTA_TOTAL_TIMEOUT)
```

Comment:
That should be covered by request's timeout (times out if the headers haven't been received in a certain time) and rsync's "data send/receive timeout" (which we have hit more than a few times).